crates/typst-library/src/foundations/version.rs

```diff
@@ -72,6 +72,15 @@ impl Version {
     /// #version((1, 2, 3, 4)) \
     /// #version((1, 2), 3)
     /// ```
+    ///
+    /// As a practical use case, it allows comparing current version
+    /// ([`{sys.version}`]($version)) to a specific one.
+    ///
+    /// // This will have to be updated in the future to keep true and false.
```

Comment:
Will this `/// // ...` here actually prevent this comment from appearing in the documentation? (if that is the intention)